Software/Rig/Rig.ino

```diff
@@ -388,6 +388,7 @@ void printSessionParameters() {
   vprint("DEBUGGING", DEBUGGING);
   vprint("IS_PRIMARY_RIG", IS_PRIMARY_RIG);
   vprint("NUMBER_OF_TRIALS", NUMBER_OF_TRIALS);
+  vprint("IS_TRAINING", IS_TRAINING);
```

Comment:
This prints the data which is set by the GUI, and the analysis pipeline could read it from the JSON